src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Intake {
+    private CANSparkMax verticalRollerMotor;
+    private CANSparkMax horizontalRollerMotor;
+    private CANSparkMax deploymentMotor;
+    private double suckSpeed;
+    private double spitSpeed;
+    private double inactiveSpeed;
+  
+    public Intake() {
+      // Initializing motor controllers
+      verticalRollerMotor = new CANSparkMax(0, MotorType.kBrushless);
+      horizontalRollerMotor = new CANSparkMax(0, MotorType.kBrushless);
+      deploymentMotor = new CANSparkMax(0, MotorType.kBrushless);
+  
+      // Motor speeds
+      suckSpeed = 1.0;
+      spitSpeed = -0.3;
+      inactiveSpeed = 0.0;
+    }
+
+    //deploy intake
+    public void deploy(boolean spit, double deploymentMotorSpeed) {
+      if (spit) {
+        verticalRollerMotor.set(spitSpeed);
+        horizontalRollerMotor.set(spitSpeed);
+      } else {
+        verticalRollerMotor.set(suckSpeed);
+        horizontalRollerMotor.set(suckSpeed);
+      }
+  
+      deploymentMotor.set(deploymentMotorSpeed);
+    }
+  
+    // stow intake
+    public void stow() {
```

Comment:
I initially thought that, but the motors should be stalled when stowed as per Trevor. 

The deployment motor gets set to inactiveSpeed in the stow method, but I believe I accidentally deleted it. I'll add it back in.

Reply:
Yeah, me and Gautam are going to hop on a call today, so that I can explain PID to him. In the meantime, I would add separate methods for running the deployment motor and intake motor (you don't want the deployment motor to be constantly running whilst you're intaking).